docs/command/query.soy

```diff
@@ -321,14 +321,14 @@ the repository and the target <code>//foo:bar</code>.
 </p>
 
 <p>
-This operator is meant to be used together with the <code>--dot</code>
+This operator is best used together with the <code>--dot</code>
 {sp}parameter to generate an image. Sample usage:
 </p>
 
-<p><pre>
+<p><pre class="prettyprint lang-py">
 {literal}
-buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
-dot -Tpng result.dot -o image.png
+$ buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
+$ dot -Tpng result.dot -o image.png
```

Comment:
No need to merge.  I can squash later.
________________________________
From: Miguel Oliveira<mailto:notifications@github.com>
Sent: ‎10/‎1/‎2015 4:09
To: facebook/buck<mailto:buck@noreply.github.com>
Cc: Shawn Wilsher<mailto:sdwilsh@fb.com>
Subject: Re: [buck] Add dot parameter description to buck query docs. (#431)


In docs/command/query.soy<https://github.com/facebook/buck/pull/431#discussion_r40901955>:

>  {literal}
> -buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
> -dot -Tpng result.dot -o image.png
> +$ buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
> +$ dot -Tpng result.dot -o image.png


Sorry for the delay. I added the $ below.

It's my first upstream PR. I had some conflicts and ended up with 5 commits. Do I need to merge them into one?

—
Reply to this email directly or view it on GitHub<https://github.com/facebook/buck/pull/431/files#r40901955>.